bouvet-deler/databricks_lambda/databricks_lambda_case2.py

```diff
@@ -0,0 +1,76 @@
+# Databricks notebook source
+# MAGIC %md
+# MAGIC # Using Lambda in Databricks - Case 2
+# MAGIC 
+# MAGIC This notebooks gives a slightly complex example of using lambda in Databricks. For more information see this article (add link).
+# MAGIC 
+# MAGIC Author: Morten Mjelde <morten.mjelde@bouvet.no>
+
+# COMMAND ----------
+
+# Define DataSet3
+from pyspark.sql import Row
+
+DataSet3 = spark.createDataFrame(
+    [
+        Row(location="Oslo", people=[Row(firstName="Harald", lastName="Haraldsen"), Row(firstName="Mark", lastName="Hunter")]),
+        Row(location="Stockholm", people=[Row(firstName="Ingrid", lastName="Hansen")]),
+    ]
+)
+
+# COMMAND ----------
+
+# Define DataSet4
+from pyspark.sql import Row
+
+DataSet4 = spark.createDataFrame(
+    [
+        Row(location="Copenhagen", people=[Row(firstName="Roger", nobleName="of house Henriksen"), Row(firstName="Hans", nobleName="of house Dale")]),
+    ]
+)
+
+# COMMAND ----------
+
+# Define transformations
+from pyspark.sql.functions import concat, col, DataFrame, lit, explode
+
+def explodePeople(df: DataFrame) -> DataFrame:
+    """Explode people column."""
+    return df.withColumn("ex_people", explode("people")).select("location", "ex_people.*")
```

Comment:
```suggestion
    return df.withColumn("expl_people", explode("people")).select("location", "expl_people.*")
```
ex_people gisves same feeling as terminated_people 😃

Reply:
Good point. Fixed.